packages/components/src/badge/tests/chromatic/Badge.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Inline, Stack } from "../../../layout/index.ts";
+import { Badge } from "../../src/Badge.tsx";
+
+const meta = {
+    title: "Components/Badge",
+    component: Badge,
+    args: {
+        children: "12"
+    }
+} satisfies Meta<typeof Badge>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Primary = {
+    render: props => (
+        <Stack>
+            <h1>Default</h1>
+            <Inline>
+                <Badge {...props} />
+            </Inline>
+            <h1>Disabled</h1>
+            <Inline>
+                <Badge {...props} isDisabled />
+            </Inline>
+        </Stack>
+    )
+} satisfies Story;
```

Comment:
add some tests with Text instead of "12"

Reply:
to ensure the text transform applies

Reply:
true, it's not just a counter anymore :P